### DISCUSSION TITLE
Talos system disk usage status

### DISCUSSION THREAD

**Discussion by barnabas-kovacs** [siderolabs/talos#9638] [Q&A] [answered]
Dear Members,

Can anybody help how to check the usage status of
the system disk of Talos where it stores the docker containers?
I did not found how I can check or monitor free space levels of it.

THX

**Comment by smira** [comment, marked as answer]
For monitoring you can use e.g. `node-exporter` (from Prometheus) like on any other Linux.

In order to inspect usage, you can use `talosctl mounts` (disk free) or `talosctl du` (disk usage).

**Reply by barnabas-kovacs** [reply]
Great Thanks!

The node exporter needs to be installed through Kubernetes
or is there any solution to install on Talos directly?

Many THX!

**Reply by smira** [reply]
node-exporter is always installed via Kubernetes
